docs/src/Tooltip.doc.js

```diff
@@ -1,165 +1,822 @@
 `}
   />,
 );
 
 card(
-  <Example
-    id="inline"
-    name="Inlining Tooltips"
-    description="Pass this flag in when rendering the tooltip around an inline element such as an Icon,
-      IconButton, or a Button with the inline prop set. This will help correctly position the tooltip
-      alongside the element."
-    defaultCode={`
-<Box>
-  <Box padding={2}>
-    <Tooltip inline text="Inline tooltip">
-      <Button
-        accessibilityLabel="This tooltip wraps an inline button and occupies the same space"
-        color="gray"
-        text="Inline"
+  <MainSection
+    name="Localization"
+    description={`If your application requires localization, be sure to localize the \`text\` and \`accessibilityLabel\` props. Note that localization can lengthen text by 20 to 30 percent.`}
+  />,
+);
+
+card(
+  <MainSection name="Variants">
+    <MainSection.Subsection
+      title="Ideal direction"
+      description={`Pass in \`idealDirection\` to specify the preferred position of Tooltip relative to its anchor element. The default direction is \`down\`, which should be used in most cases. The actual position may change given the available space around the anchor element.`}
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+        function ExampleTooltip() {
+  const [idealDirection, setIdealDirection] = React.useState('down');
+
+  return (
+    <Flex direction="column" alignItems="center" gap={8}>
+      <Flex gap={4}>
+        <RadioButton
+          checked={idealDirection === 'up'}
+          id="up"
+          label="Up"
+          name="idealDirection"
+          onChange={() => setIdealDirection( 'up' )}
+          value="up"
+        />
+        <RadioButton
+          checked={idealDirection === 'right'}
+          id="right"
+          label="Right"
+          name="idealDirection"
+          onChange={() => setIdealDirection( 'right' )}
+          value="right"
+        />
+        <RadioButton
+          checked={idealDirection === 'down'}
+          id="down"
+          label="Down"
+          name="idealDirection"
+          onChange={() => setIdealDirection( 'down' )}
+          value="down"
+        />
+        <RadioButton
+          checked={idealDirection === 'left'}
+          id="left"
+          label="Left"
+          name="idealDirection"
+          onChange={() => setIdealDirection( 'left' )}
+          value="left"
+        />
+      </Flex>
+      <Tooltip
+        idealDirection={idealDirection}
         inline
+        text="Share"
+        >
+          <IconButton
+            accessibilityLabel="Share this Pin"
+            bgColor="white"
+            icon="share"
+            iconColor="darkGray"
+            inline
+            size="lg"
+          />
+        </Tooltip>
+      </Flex>
+
+  )
+}
+`}
       />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Inline"
```

Comment:
What did you think about using this example from the product for inline?
![Screen Shot 2021-02-12 at 10 45 51 AM](https://user-images.githubusercontent.com/5125094/108108458-80269a80-7045-11eb-8873-6e0f5d6a83e6.png)

Reply:
+1, that's a good demonstration of the inline aspect

Reply:
@dangerismycat  @ayeshakmaz  Believe, I wasn't being lazy with this example ahah inline ain't needed with Flex, jsutifyContent='end'... I tried 3-4 different cases and couldn't come up with a good example that made sense. I decided to go with a basic button. I might need help with this one if we really want to add a different example.

Reply:
Ha, we believe you. 🙂 One use case I can think of off the top of my head is when a button is at the end of a potentially long string of text, and it should remain at the end of the last line text instead of next to the block of text. I recall that coming up with Board/Profile headers. Perhaps you can replicate something similar?